lib/index.js

```diff
@@ -1190,6 +1196,10 @@ exposeMethod('consume');
  * xs.pull(function (err, x) {
  *     // do something
  * });
+ *
+ * // Creating a promise with all the values using Q
+ * q.ninvoke(xs.collect(), "pull")
+ *   .then(function (values) {console.log(values)})
```

Comment:
You can do `.then(_.log)` here. It's better to use existing functions than to roll your own anonymous functions all the time.